shared_stack.bash

```diff
@@ -50,10 +50,10 @@ ROOT=/scratch/rubinsw
 # State directory
 STATE=~/shared-stack
 # Top-level products to install
-PRODUCTS="lsst_sitcom"
+PRODUCTS="lsst_sitcom lsst_distrib"
 # Number of daily releases to keep
 # shellcheck disable=SC2034
-KEEP_d=24
+KEEP_d=48
```

Comment:
What's the use case for 48 dailies?

Reply:
2 months of history (6 a week).  For A/B comparisons in case something changed.